src/test/e2e/35_custom_init_package_test.go

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCustomInit(t *testing.T) {
+	t.Log("E2E: Custom Init Package")
+	e2e.SetupWithCluster(t)
+	buildPath := filepath.Join("src", "test", "packages", "35-custom-init-package")
+	pkgName := fmt.Sprintf("build/zarf-init-%s-%s.tar.zst", e2e.Arch, e2e.GetZarfVersion(t))
+	privateKeyFlag := "--signing-key=src/test/packages/zarf-test.prv-key"
+	publicKeyFlag := "--key=src/test/packages/zarf-test.pub"
+
+	stdOut, stdErr, err := e2e.Zarf("package", "create", buildPath, privateKeyFlag, "--confirm", "-o=build")
```

Comment:
Putting this in the `build` directory will cause a failure with the `k3s` test because it will overwrite the init package that was used to deploy it causing this error: https://github.com/defenseunicorns/zarf/actions/runs/7700470953/job/20984462726?pr=2259#step:7:15068

(this should just be placed in the working directory instead (Zarf will prioritize that file over the one that is next to it's binary)

Reply:
Ok, I think I understand what you are saying.  I pushed an update, let me know if this is what you meant.